### PR TITLE
[FR] Add `Wiki/Tournament_drawings`

### PR DESCRIPTION
**Pull request by Biokax**
## Self-check

the last translation of english outdated article

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Biokax** [comment]
Since its based on an outdated article, does the screenshot needs to be translated too ?

**Comment by Walavouchey** [comment]
the lazer tournament client doesn't have localisation if i'm not mistaken, but other than that i suppose there's no need to bother

**Comment by Biokax** [comment]
> the lazer tournament client doesn't have localisation if i'm not mistaken, but other than that i suppose there's no need to bother

Lazer client have localisation based on crowdin if im not wrong, I will check it later

**Comment by Walavouchey** [comment]
i said the "lazer tournament client". i checked in crowdin and there are no strings for the tournament client for what i can see

File: wiki/Tournament_drawings/fr.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+drawings.txt
+```
+
+Ensuite, ajoutez la ligne suivante au fichier `drawings.txt` :
+
+```
+AU : Australia : AUS
+```
+
+Les tournament drawings, sont maintenant prêt à être prévisualisés. Il est important de comprendre comment l'écran pour les tournament drawings fonctionne et agit avant de modifier d'avantage chacun des fichiers.
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:26]:
```suggestion
Les tournament drawings sont maintenant prêt à être prévisualisés. Il est important de comprendre comment l'écran des tournament drawings fonctionne et agit avant de modifier davantage chacun des fichiers.  
```

File: wiki/Tournament_drawings/fr.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+```
+AU : Australia : AUS
+```
+
+Les tournament drawings, sont maintenant prêt à être prévisualisés. Il est important de comprendre comment l'écran pour les tournament drawings fonctionne et agit avant de modifier d'avantage chacun des fichiers.  
+
+### Utilisation
+
+À partir du menu principal, appuyer sur `Ctrl + Shift + D` afin d'accéder à l'écran de dessins de tournois. L'écran suivant devrait apparaître :
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:30]:
```suggestion
À partir du menu principal, appuyez sur `Ctrl + Shift + D` afin d'accéder à l'écran des tournament drawings. L'écran suivant devrait apparaître :
```

File: wiki/Tournament_drawings/fr.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+![](img/main-screen.png "Le menu de dessins de tournois")
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:31]:
```suggestion
![](img/main-screen.png "Menu principal des tournament drawings")
```

File: wiki/Tournament_drawings/fr.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+Cet écran est composé de deux sections - la partie principale se situant sur la gauche, avec la carte du monde en tant que fond d'écran, et la panneau de sélection des contrôles sur la droite. **Évitez** de diffuser en direct ce panneau de contrôle.
+
+Il y a 4 boutons sur ce panneau de contrôle, passons-les en revue :
+
+- **Begin random**
+  - Ceci va commencer le procédé aléatoire, faisant défiler les drapeaux de l'écran.
+- **Stop random**
+  - Ceci va arrêter le procédé aléatoire, faisant ralentir le défilement des drapeaux, jusqu'à ce que qu'il s'arrête éventuellement sur le drapeau au centre de l'écran
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:40]:
```suggestion
  - Ceci va arrêter le procédé aléatoire, faisant ralentir le défilement des drapeaux, jusqu'à ce que qu'il s'arrête sur un drapeau au centre de l'écran.
```

File: wiki/Tournament_drawings/fr.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+Il y a 4 boutons sur ce panneau de contrôle, passons-les en revue :
+
+- **Begin random**
+  - Ceci va commencer le procédé aléatoire, faisant défiler les drapeaux de l'écran.
+- **Stop random**
+  - Ceci va arrêter le procédé aléatoire, faisant ralentir le défilement des drapeaux, jusqu'à ce que qu'il s'arrête éventuellement sur le drapeau au centre de l'écran 
+- **Reload**
+  - Ceci va recharger le fichier `drawings.txt`.
+- **Reset**
+  - Ce bouton doit rarement être utilisé. Il est destructif, et va réinitialiser les résultats du procédé de dessins.
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:44]:
```suggestion
  - Ce bouton doit rarement être utilisé. Il est destructif, et va réinitialiser les résultats du procédé.
```

File: wiki/Tournament_drawings/fr.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
+---
+outdated: true
+---
+
+# Tournament drawings
+
+L'écran des tournament drawings est utilisé pour diffuser en direct le tri des équipes en groupes qui s'affronteront lors des phases de groupes d'un tournoi. Il est uniquement disponible dans le client [osu! (lazer)](https://github.com/ppy/osu/releases).
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:7]:
```suggestion
L'écran des tournament drawings est utilisé pour diffuser en direct le tirage des équipes en groupes qui s'affronteront lors des phases de groupes d'un tournoi. Il est uniquement disponible dans le client [osu! (lazer)](https://github.com/ppy/osu/releases).
```

File: wiki/Tournament_drawings/fr.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+- **Stop random**
+  - Ceci va arrêter le procédé aléatoire, faisant ralentir le défilement des drapeaux, jusqu'à ce que qu'il s'arrête éventuellement sur le drapeau au centre de l'écran 
+- **Reload**
+  - Ceci va recharger le fichier `drawings.txt`.
+- **Reset**
+  - Ce bouton doit rarement être utilisé. Il est destructif, et va réinitialiser les résultats du procédé de dessins.
+
+Appuyez sur les boutons `Begin random` et `Stop random`. Lorsque le défilement s'arrête sur un drapeau au centre de l'écran, retournez dans le dossier osu! et remarquez qu'un fichier supplémentaire a été créé, appelé `drawings_results.txt`.
+
+Ouvrez le fichier et observez le format - c'est là que sont stockés les résultats du processus de dessin, et ils doivent être importés dans d'autres outils d'aide à la gestion de tournois, tels que Google Spreadsheets.\
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:48]:
```suggestion
Ouvrez le fichier et observez le format - c'est là que sont stockés les résultats du processus de tirage, et ils doivent être importés dans d'autres outils d'aide à la gestion de tournois, tels que Google Spreadsheets.\
```

File: wiki/Tournament_drawings/fr.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+
+Un tournoi ne peut pas être composé de 8 groupes, et de même chaque groupe ne peut pas être composé de 8 équipes. Le fichier `drawings.ini` est un fichier de configuration qui permet l'ajustement de certaines propriétés.
+Un fichier de configuration valide ressemble à :
+
+```
+Groups = 4
+TeamsPerGroup = 4
+```
+
+Les propriétés suivantes sont des propriétés configurables au travers de ce fichier :
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:62]:
```suggestion
Les propriétés suivantes sont des propriétés configurables en utilisant ce fichier :
```

File: wiki/Tournament_drawings/fr.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+
+| Propriété | Description | Valeur Valide | Valeur Par Défaut |
+| :-- | :-- | :-- | :-- |
+| Groups | Le nombre de groupe dans lequel on va trier les équipes. | Entre 1 et 8 (inclusif) | 8 |
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:66]:
```suggestion
| Groups | Le nombre de groupes dans lesquels les équipes seront triées. | Entre 1 et 8 (inclus) | 8 |
```

File: wiki/Tournament_drawings/fr.md [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+| TeamsPerGroup | Le nombre maximum d'équipes au sein d'un même groupe. | Entre 1 et 8 (inclusif) | 8 |
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:67]:
```suggestion
| TeamsPerGroup | Le nombre maximum d'équipes au sein d'un même groupe. | Entre 1 et 8 (inclus) | 8 |
```

File: wiki/Tournament_drawings/fr.md [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+
+### Définition des équipes
+
+Le fichier `drawings.txt` fourni une méthode de spécification des équipes qui vont être triées en groupes sur des lignes séparées. Voici une ligne d'exemple :
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:71]:
```suggestion
Le fichier `drawings.txt` fournit une méthode de spécification des équipes qui vont être triées en groupes sur des lignes séparées. Voici une ligne d'exemple :
```

File: wiki/Tournament_drawings/fr.md [hunk 12]
```diff
@@ -0,0 +1,112 @@
...
+
+```
+AU : Australia : AUS
+```
+
+Cette ligne possède trois sections séparées par une colonne (`:`) :
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:77]:
```suggestion
Cette ligne possède trois sections séparées par un double point (`:`) :
```

File: wiki/Tournament_drawings/fr.md [hunk 13]
```diff
@@ -0,0 +1,112 @@
...
+
+| Nom Du Drapeau | Nom Complet De L'équipe | Acronyme De L'équipe |
+| :-: | :-: | :-: |
+| AU | Australia | AUS |
+
+- Le nom du drapeau fait référence au nom du fichier qui fourni l'image du drapeau. Par défaut, osu! (lazer) fourni les drapeaux de pays en [Code Pays ISO 3166 Alpha-2](https://www.iso.org/iso-3166-country-codes.html).
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:83]:
```suggestion
- Le nom du drapeau fait référence au nom du fichier qui fournit l'image du drapeau. Par défaut, osu! (lazer) fournit les drapeaux de pays en [Code Pays ISO 3166 Alpha-2](https://www.iso.org/iso-3166-country-codes.html).
```

File: wiki/Tournament_drawings/fr.md [hunk 14]
```diff
@@ -0,0 +1,112 @@
...
+- Le nom complet de l'équipe est affiché au centre de l'écran quand une équipe est sélectionnée au travers du procédé de défilement.
+- L'acronyme de l'équipe est affiché dans les case de groupe.
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:85]:
```suggestion
- L'acronyme de l'équipe est affiché dans les cases de groupe.
```

File: wiki/Tournament_drawings/fr.md [hunk 15]
```diff
@@ -0,0 +1,112 @@
...
+
+```
+my-flag-file : My Team : MT
+```
+
+La résolution pour une image de drapeau optimale est de 70x47 pixels (ou le même rapport d'aspect)
+
+### Seeding
+
+Il peut-être désirable de "seed" des équipes. Dans ce cas, de multiples fichiers `drawings.txt` peuvent-être "hotswapped" (remplacé à chaud) à l'aid du bouton de Rechargement depuis le panneau de contrôle.
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:108]:
```suggestion
Il peut être désirable de "seed" des équipes. Dans ce cas, de multiples fichiers `drawings.txt` peuvent être intervertis à l'aide du bouton Reload depuis le panneau de contrôle.
```

File: wiki/Tournament_drawings/fr.md [hunk 16]
```diff
@@ -0,0 +1,112 @@
...
+Un fichier `drawings.txt` valide avec à l'intérieur plusieurs pays en tant qu'équipes :
+
+```
+AU : Australia : AUS
+RO : Romania : RO
+IT : Italy : IT
+US : United States of America : USA
+```
+
+Pour définir un drapeau personnalisé, retournez dans le dossier osu! contenant le fichier `drawings.ini`, à l'intérieur vous trouverez le dossier `Flags`. Les images de drapeaux personnalisées peuvent-être placés à l'intérieur du dossier Flags. Par exemple, si le fichier `my-flag-file.png` a été placé à l'intérieur de ce dossier, alors une ligne valide pouvant être ajoutée au fichier `drawings.txt` se présente sous la forme de :
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:98]:
```suggestion
Pour définir un drapeau personnalisé, retournez dans le dossier osu! contenant le fichier `drawings.ini`, à l'intérieur vous trouverez le dossier `Flags`. Les images de drapeaux personnalisées peuvent-être placées à l'intérieur du dossier Flags. Par exemple, si le fichier `my-flag-file.png` a été placé à l'intérieur de ce dossier, alors une ligne valide pouvant être ajoutée au fichier `drawings.txt` se présente sous la forme de :
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Tournament_drawings/fr.md [hunk 17]
```diff
@@ -0,0 +1,112 @@
...
+```
+
+Les tournament drawings sont maintenant prêt à être prévisualisés. Il est important de comprendre comment l'écran des tournament drawings fonctionne et agit avant de modifier davantage chacun des fichiers.  
+
+### Utilisation
+
+À partir du menu principal, appuyez sur `Ctrl + Shift + D` afin d'accéder à l'écran des tournament drawings. L'écran suivant devrait apparaître :
+![](img/main-screen.png "Menu principal des tournament drawings")
+
+Cet écran est composé de deux sections - la partie principale se situant sur la gauche, avec la carte du monde en tant que fond d'écran, et la panneau de sélection des contrôles sur la droite. **Évitez** de diffuser en direct ce panneau de contrôle.
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:33]:
```suggestion
Cet écran est composé de deux sections - la partie principale se situant sur la gauche, avec la carte du monde en tant que fond d'écran, et le panneau de sélection des contrôles sur la droite. **Évitez** de diffuser en direct ce panneau de contrôle.
```

File: wiki/Tournament_drawings/fr.md [hunk 18]
```diff
@@ -0,0 +1,112 @@
...
+Un fichier `drawings.txt` valide avec à l'intérieur plusieurs pays en tant qu'équipes :
+
+```
+AU : Australia : AUS
+RO : Romania : RO
+IT : Italy : IT
+US : United States of America : USA
+```
+
+Pour définir un drapeau personnalisé, retournez dans le dossier osu! contenant le fichier `drawings.ini`, à l'intérieur vous trouverez le dossier `Flags`. Les images de drapeaux personnalisées peuvent-être placées à l'intérieur du dossier Flags. Par exemple, si le fichier `my-flag-file.png` a été placé à l'intérieur de ce dossier, alors une ligne valide pouvant être ajoutée au fichier `drawings.txt` se présente sous la forme de :
```
Comment (Ganondorf-1) [inline wiki/Tournament_drawings/fr.md:98]:
```suggestion
Pour définir un drapeau personnalisé, retournez dans le dossier osu! contenant le fichier `drawings.ini`, vous y trouverez le dossier `Flags`. Les images de drapeaux personnalisées peuvent-être placées à l'intérieur du dossier Flags. Par exemple, si le fichier `my-flag-file.png` a été placé à l'intérieur de ce dossier, alors une ligne valide pouvant être ajoutée au fichier `drawings.txt` se présente sous la forme de :
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Tournament_drawings/fr.md [hunk 19]
```diff
@@ -0,0 +1,112 @@
...
+US : United States of America : USA
+```
+
+Pour définir un drapeau personnalisé, retournez dans le dossier osu! contenant le fichier `drawings.ini`, vous y trouverez le dossier `Flags`. Les images de drapeaux personnalisées peuvent-être placées à l'intérieur du dossier Flags. Par exemple, si le fichier `my-flag-file.png` a été placé à l'intérieur de ce dossier, alors une ligne valide pouvant être ajoutée au fichier `drawings.txt` se présente sous la forme de :
+
+```
+my-flag-file : My Team : MT
+```
+
+La résolution pour une image de drapeau optimale est de 70x47 pixels (ou le même rapport d'aspect)
```
Comment (Biokax) [inline wiki/Tournament_drawings/fr.md:104]:
```suggestion
La résolution pour une image de drapeau optimale est de 70x47 pixels (ou le même rapport d'aspect).
```

**Review by Ganondorf-1 — APPROVED** [review]

File: wiki/Tournament_drawings/fr.md [hunk 20]
```diff
@@ -0,0 +1,112 @@
...
+my-flag-file : My Team : MT
+```
+
+La résolution pour une image de drapeau optimale est de 70x47 pixels (ou le même rapport d'aspect).
+
+### Seeding
+
+Il peut être désirable de "seed" des équipes. Dans ce cas, de multiples fichiers `drawings.txt` peuvent être intervertis à l'aide du bouton Reload depuis le panneau de contrôle.
+
+## Questions?
```
Comment (Biokax) [inline wiki/Tournament_drawings/fr.md:110]:
```suggestion
## Questions ?
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**